fluxfw-gcp/img/login.pkr.hcl

```diff
@@ -12,6 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+variable "enable_secure_boot" {
```

Comment:
This seems like a significant change. Other than upping the impression of "security" what is the goal with adding this?

Reply:
It saves you having to fiddle with the Argolis policy that requires shielded VMs.